src/login/logind-user.c

```diff
@@ -541,7 +541,7 @@ static int user_remove_runtime_path(User *u) {
 
         r = rm_rf(u->runtime_path, 0);
         if (r < 0)
-                log_error_errno(r, "Failed to remove runtime directory %s: %m", u->runtime_path);
+                log_error_errno(r, "Failed to remove runtime directory %s (before unmounting): %m", u->runtime_path);
```

Comment:
maybe similarly check the message of the other rm_rf() call? that way things are easier to debug should users report issues with old vs. new logind versions